components/coding/profileV2/ProfileHeaderComponent.tsx

```diff
@@ -83,12 +83,14 @@ export default function UserProfileSection({ user }: UserProfileSectionProps) {
               <JoinedDateComponent
                 user={user}
                 createdAt={userProfileData.createdAt}
+                textSize={"large"}
               />
             </div>
             <BadgesDisplayedComponent
               user={user}
               earnedBadges={userBadgeCount}
               totalBadges={badgeCount}
+              textSize={"large"}
```

Comment:
great idea to put a prop in :) this will be super useful so we don't have to keep creating new components for different use cases.